test/TestProjects/ModelWithConverterUsage/Generated/Models/ModelStruct.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Text.Json.Serialization;
+
+namespace ModelWithConverterUsage.Models
+{
+    /// <summary> The ModelStruct. </summary>
+    [JsonConverter(typeof(ModelStructConverter))]
```

Comment:
Should we move this attribute to the other part of the file? The same where ModelStructConverter is?

Reply:
Yeah, this looks better.